app/javascript/onboarding/Onboarding.jsx

```diff
@@ -69,3 +71,7 @@ export default class Onboarding extends Component {
     return <main className="onboarding-body">{this.slides[currentSlide]}</main>;
   }
 }
+
+Onboarding.propTypes = {
+  communityConfig: PropTypes.shape.isRequired,
```

Comment:
```suggestion
  communityConfig: PropTypes.shape({
    communityName: PropTypes.string.isRequired,
    communityDescription: PropTypes.string.isRequired
  })
```
TIL: about `PropTypes.shape`! Looking at [the docs](https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes), I think we should be specifying the keys in the object, right?